portfolio/src/main/webapp/style.css

```diff
@@ -61,6 +61,26 @@ body {
   padding: 0.5em 0.5em 0.5em 0.5em;
 }
 
+.comments-card {
+  background-color: #ffffff;
```

Comment:
Use color name instead of hex value when possible

Reply:
Done